message tab_ScoreRuleTable 
{
	optional string tname = 1;
	repeated tab_ScoreRule tlist = 2;
}

message tab_ScoreRule
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 ScoreAddPerPoint = 3;
	optional int32 ScoreAddWin = 4;
	optional int32 CoinsAddWin = 5;
	optional int32 ScoreSubPerPoint = 6;
	optional int32 ScoreSubLose = 7;
	optional int32 CoinsSubLose = 8;
}